web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -178,9 +182,16 @@ public String getFullRecord(String pidString, Model model, HttpServletRequest re
             throw new InvalidRecordRequestException();
         }
 
-        // Retrieve the objects record from Solr
-        SimpleIdRequest idRequest = new SimpleIdRequest(pid, principals);
-        ContentObjectRecord briefObject = queryLayer.getObjectById(idRequest);
+        // Retrieve the object's record from Solr
+        SearchRequest searchRequest = new SearchRequest();
```

Comment:
I don't think I understand why this change was made for how the object's record itself is retrieved?

Reply:
I think I just borrowed the syntax from another class. I can switch it back.